### DISCUSSION TITLE
Lucid to draw.io Chrome extension does not work

### DISCUSSION THREAD

**Discussion by nitints** [jgraph/drawio#3285] [Q&A]
Hello

trying to use the chrome extension to import Lucid diagrams into draw.io but it doesn't work
 Error: Couldn't fetch documents list.

<img width="503" alt="image" src="https://user-images.githubusercontent.com/60752877/212398689-7dc918f1-4d3a-44b9-8449-d5c2d2972a5f.png">

Hoping the bug can be fixed, as we are looking forward to moving all users to diagrams.net

**Comment by samhodgeman** [comment]
I see this question is open since January. I have the same issue. Are there no updates? Really need this bulk import capability.

**Comment by karolba** [comment]
Seems like whatever it was it's been fixed in those years since the following comments, cause we've successfully used the extension now
